src/elements/stepper/stepper/readme.md

```diff
@@ -0,0 +1,111 @@
+The `sbb-stepper` is a component that visually guides a user through a sequential, multi-step process. It breaks down complex forms, flows, or other linear interactions into smaller, easier-to-follow steps. The current step is highlighted, and a progress bar connects the steps to visually represent progress.
+
+Use it with [sbb-step-label](/docs/components-sbb-stepper-sbb-step-label--docs) and [sbb-step](/docs/components-sbb-stepper-sbb-step--docs).
+
+```html
+<sbb-stepper aria-label="Purpose of this flow">
+  <sbb-step-label>Step label 1</sbb-step-label>
+  <sbb-step>Step content 1</sbb-step>
+
+  <sbb-step-label>Step label 2</sbb-step-label>
+  <sbb-step>Step content 2</sbb-step>
+</sbb-stepper>
+```
+
+## Interactions
+
+There are two attributes to support navigation between different steps that can be used on elements inside an `sbb-step` to select the next or the previous step when clicked: `sbb-stepper-next` and `sbb-stepper-previous`.
+
+### Linear stepper
+
+The `linear` property can be set to create a linear stepper that requires the user to complete previous steps before proceeding to following steps.
+
+```html
+<sbb-stepper aria-label="Purpose of this flow" linear>
+  <sbb-step-label>Step label 1</sbb-step-label>
+  <sbb-step>Step content 1</sbb-step>
+
+  <sbb-step-label>Step label 2</sbb-step-label>
+  <sbb-step>Step content 2</sbb-step>
+
+  <sbb-step-label>Step label 3</sbb-step-label>
+  <sbb-step>Step content 3</sbb-step>
+</sbb-stepper>
+```
+
+## Forms
+
+There are two possible approaches. One is using a single form for the stepper, and the other is using a different form for each step.
+
+### Single form
+
+```html
+<form>
+  <sbb-stepper aria-label="Purpose of this flow">
+    <sbb-step-label>Step label 1</sbb-step-label>
+    <sbb-step>Step content 1: <sbb-form-field>...</sbb-form-field></sbb-step>
+
+    <sbb-step-label>Step label 2</sbb-step-label>
+    <sbb-step>Step content 2: <sbb-form-field>...</sbb-form-field></sbb-step>
+  </sbb-stepper>
+</form>
+```
+
+### Multiple forms
+
+```html
+<sbb-stepper aria-label="Purpose of this flow">
+  <sbb-step-label>Step label 1</sbb-step-label>
+  <sbb-step>
+    <form>
+      <sbb-form-field>...</sbb-form-field>
+    </form>
+  </sbb-step>
+
+  <sbb-step-label>Step label 2</sbb-step-label>
+  <sbb-step>
+    <form>
+      <sbb-form-field>...</sbb-form-field>
+    </form>
+  </sbb-step>
+</sbb-stepper>
+```
+
+Calling the `reset()` method on the `sbb-stepper`, will reset the wrapping form or every form of each step and select the first step.
```

Comment:
```suggestion
Calling the `reset()` method on the `sbb-stepper` will reset the wrapping `form` or, if they are present, every `form` in each step; then it will select the first step.
```